examples/aws/verify_egressv1.go

```diff
@@ -6,9 +6,9 @@ import (
 	"os"
 	"time"
 
-	"github.com/aws/aws-sdk-go/aws/credentials"
-	ocmlog "github.com/openshift-online/ocm-sdk-go/logging"
-	"github.com/openshift/osd-network-verifier/pkg/cloudclient"
+	"github.com/aws/aws-sdk-go-v2/credentials"
```

Comment:
given we're using `github.com/aws/aws-sdk-go-v2/credentials` I can recommend changing the filename as `verify_egressv2.go`